### DISCUSSION TITLE
[beta] Please move Saved Filters dropdown field to Results tab

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#11016] [Q&A]
Hi, I tried the current v3.4-beta1 for the last few days and have been very excited about the new Saved Filters feature. I noticed that the dropdown field for choosing a saved filter resides on the Filter tab, which seems logical. However, it would be easier and faster to use if the drop-down field were placed on the Results tab next to Quick Search.

What do you think?

BTW: I am not able to enter the beta demo with admin:admin at [https://beta-demo.netbox.dev](https://beta-demo.netbox.dev) at the moment. "Please enter a correct username and password" it says. Is this intended?

**Comment by ghost** [comment]
YES! Please have saved filters in both locations. It's great that its in the filters tab, which I agree makes sense.
But if I want to quickly select a saved filter why is it 4 clicks?
Click 1 - filters tab
Click 2 - saved filters dropdown
Click 3 - select saved filter
Click 4 - click search

@jeremystretch , is it better to talk about this specifically here in discussions? Or would it need a FR submitted where it can be discussed there?
